src/components/Project/Project.js

```diff
@@ -9,7 +8,7 @@ const Project = (props) => {
         <h2 className="proj-name">
           {props.project_title}
         </h2>
-        <div className="saved-palette-disp">
+        <div className="saved-palette-disp" onClick={() => props.editPalette(id)}>
```

Comment:
Passing the id as discussed works well because we are properly controlling how we edit a single palette.